flow/project.py

```diff
@@ -369,12 +512,12 @@ def cmd(self):
 
     def set_status(self, value):
         "Store the operation's status."
-        self.job._project.document.setdefault('_status', dict())[self.id] = int(value)
+        self.jobs[0]._project.document.setdefault('_status', dict())[self.id] = int(value)
 
     def get_status(self):
         "Retrieve the operation's last known status."
         try:
-            return JobStatus(self.job._project.document['_status'][self.id])
+            return JobStatus(self.jobs[0]._project.document['_status'][self.id])
```

Comment:
See comment above for `set_status`. The problem comes in that if the number of jobs change in between status checks, the jobs grouped together will change, and this will result in different ids.